docs/ml/index.asciidoc

```diff
@@ -56,17 +57,17 @@ occurring in your operational environment at that time:
 image::ml/images/ml-annotations-list.jpg[Single Metric Viewer with annotations]
 
 In some circumstances, annotations are also added automatically. For example, if
-the {ml} analytics detect that there is missing data, it annotates the affected
+the {anomaly-job} detects that there is missing data, it annotates the affected 
 time period. For more information, see 
-{stack-ov}/ml-delayed-data-detection.html[Handling delayed data].
-The *Job Management* pane shows the full list of annotations for each job.
+{stack-ov}/ml-delayed-data-detection.html[Handling delayed data]. The 
+*Job Management* pane shows the full list of annotations for each job.
 
-NOTE: The {kib} {ml-features} use pop-ups. You must configure your
-web browser so that it does not block pop-up windows or create an exception for
-your {kib} URL.
+NOTE: The {kib} {ml} {anomaly-detect} use pop-ups. You must configure your web 
```

Comment:
I think this change should be reverted too.  I think all the tabs in the ML app in Kibana potentially use pop-ups so it's better to leave this as-is.